app/lib/theme/widgets/ReadOnlyCalculatedValueWidget.tsx

```diff
@@ -14,9 +14,18 @@ const ReadOnlyCalculatedValueWidget: React.FC<WidgetProps> = ({
   const calculatedValue =
     formContext[uiSchema.calculatedValueFormContextProperty];
 
+  const selectMessageIfNullProperty = (schemaProperty) => {
+    switch (schemaProperty) {
+      case "calculatedRank":
+        return "Please enter a score in the above field to see the ranking of this project comparing to other ones with scores entered.";
+      default:
+        return "";
+    }
+  };
+
   return (
     <>
-      <dd aria-label={label}>
+      <div aria-label={label}>
```

Comment:
Is `div` correct here? The `dd` throws accessibility violations (https://dequeuniversity.com/rules/axe/4.1/dlitem)

Reply:
I created a `CalculatedValueWidget` to address this and used it in the `RankWidget`